docs/Contributing/API-for-contributors.md

```diff
@@ -2095,6 +2096,40 @@ This replaces the active global enroll secrets with the secrets specified.
 
 `Status: 200`
 
+### Store secret variables
+
+Stores secret variables prefixed with `$FLEET_SECRET_` to Fleet.
+
+`POST /api/v1/fleet/spec/secret_variables`
+
+#### Parameters
+
+| Name    | Type | In   | Description                                                                                                      |
+| ------- | ---- | ---- | ---------------------------------------------------------------------------------------------------------------- |
+| secrets | list | body | **Required.** List of objects consists of a `name` and `value` fields. |
```

Comment:
This means the 1 Fleet server can only have 1 value for a secret. Theoretically, you could have a different value per team. In this case, we would simply save the latest value.

Also, what if a user added a script/profiile directly via the API with FLEET_SECRET value? Would we allow them to use a secret which was already saved in Fleet?

Reply:
> This means the 1 Fleet server can only have 1 value for a secret. Theoretically, you could have a different value per team. In this case, we would simply save the latest value.

@getvictor We discussed this during the design review and we think it's ok if we can add only one value for a secret globally. This covers the use case for the customer.

> Also, what if a user added a script/profiile directly via the API with FLEET_SECRET value? Would we allow them to use a secret which was already saved in Fleet?

We want to allow users to use $FLEET_SECRET if it exist in Fleet via UI or API. If it doesn't exist we want to error. WIll add errors to Figma.